action/action-simulator/src/main/java/com/powsybl/action/simulator/tools/ActionSimulatorTool.java

```diff
@@ -54,6 +54,7 @@
     private static final String CASE_FILE = "case-file";
     private static final String DSL_FILE = "dsl-file";
     private static final String CONTINGENCIES = "contingencies";
+    private static final String APPLY_IF_WORKS = "apply-if-works";
```

Comment:
I’m not sure that this parameter name is clear enough. It seems that if this parameter is true, you don’t test the other actions.

Reply:
Yes, if this option is activated, and there are no violations in LF's result by applying one of test actions, the simulation of contingency stops.